cointop/table_header.go

```diff
@@ -217,12 +217,12 @@ func (ct *Cointop) UpdateTableHeader() error {
 		arrow := " "
 		colorfn := baseColor
 		if !noSort {
-			if ct.State.sortBy == col {
+			corSortCons := ct.State.viewSorts[ct.State.selectedView]
```

Comment:
do you mean cur (current)?

Reply:
Yes, "current" is true, I've changed the var name.